isis/tests/FunctionalTestsIsis2Std.cpp

```diff
@@ -0,0 +1,1054 @@
+                           "green=" + inputCubeFilename + "+4",
+                           "blue=" + inputCubeFilename + "+4",
+                           "to=" + outputPNGFilename,
+                           "mode=rgb",
+                           "format=png",
+                           "stretch=manual",
+                           "rmin=0",
+                           "rmax=2",
+                           "gmin=1",
+                           "gmax=3",
+                           "bmin=2",
+                           "bmax=6"};
+
+  UserInterface options(APP_XML, args);
+  try {
+    isis2std(options);
+  }
+  catch (IException &e) {
+    FAIL() << "Unable to translate image: " << e.what() << std::endl;
+  }
+
+  QString reingestCubeFilename = tempDir.path() + "/test_output.cub";
+  QVector<QString> reingestArgs = {"from=" + outputPNGFilename,
+                                   "to=" + reingestCubeFilename};
+
+  UserInterface reingestOptions(STD2ISIS_XML, reingestArgs);
+  try {
+    std2isis(reingestOptions);
+  }
+  catch (IException &e) {
+    FAIL() << "Unable to reingest image: " << e.what() << std::endl;
+  }
+
+  Cube reingestCube(reingestCubeFilename);
+  inputCube.open(inputCubeFilename);
+  int pixelsPerBand = inputCube.lineCount() * inputCube.sampleCount();
+
+  std::unique_ptr<Histogram> redHist(reingestCube.histogram(1));
+  std::unique_ptr<Histogram> greenHist(reingestCube.histogram(2));
+  std::unique_ptr<Histogram> blueHist(reingestCube.histogram(3));
+
+  EXPECT_EQ(redHist->Minimum(), 128);
+  EXPECT_EQ(redHist->Maximum(), 255);
+  EXPECT_EQ(redHist->ValidPixels(), pixelsPerBand);
+  EXPECT_EQ(redHist->TotalPixels(), pixelsPerBand);
+  EXPECT_EQ(redHist->Average(), 223.25);
+  EXPECT_NEAR(redHist->StandardDeviation(), 55.223, 0.001);
+
+  EXPECT_EQ(greenHist->Minimum(), 1);
+  EXPECT_EQ(greenHist->Maximum(), 255);
+  EXPECT_EQ(greenHist->ValidPixels(), pixelsPerBand);
+  EXPECT_EQ(greenHist->TotalPixels(), pixelsPerBand);
+  EXPECT_EQ(greenHist->Average(), 159.75);
+  EXPECT_NEAR(greenHist->StandardDeviation(), 105.744, 0.001);
+
+  EXPECT_EQ(blueHist->Minimum(), 1);
+  EXPECT_EQ(blueHist->Maximum(), 128);
+  EXPECT_EQ(blueHist->ValidPixels(), pixelsPerBand);
+  EXPECT_EQ(blueHist->TotalPixels(), pixelsPerBand);
+  EXPECT_EQ(blueHist->Average(), 48.5);
+  EXPECT_NEAR(blueHist->StandardDeviation(), 52.835, 0.001);
+}
+
+
+TEST_F(IsisTruthCube, FunctionalTestsIsis2StdExtension) {
+  QString outputBmpFilename = tempDir.path() + "/test_output";
+  QVector<QString> args = {"from=" + inputCubeFilename,
+                           "to=" + outputBmpFilename,
+                           "mode=grayscale",
+                           "format=bmp",
+                           "stretch=linear"};
+
+  UserInterface options(APP_XML, args);
+  try {
+    isis2std(options);
+  }
+  catch (IException &e) {
+    FAIL() << "Unable to translate image: " << e.what() << std::endl;
+  }
+
+  QFile outputFile(outputBmpFilename + ".bmp");
```

Comment:
We could test this for all of the other extensions, but this is all the old tests did so I just went with this. This app already has plenty of tests.